### PR TITLE
MGMT-11449: govc installation from GitHub gets rate-limiter failures

### PR DESCRIPTION
**Pull request by adriengentil**
Install govc from the container registry instead of relying on github.

### CODE REVIEW

**Comment by adriengentil** [comment]
/hold

until the mapping in openshift/release is done.

File: Dockerfile.assisted-test-infra [hunk 1]
```diff
@@ -38,6 +37,8 @@ RUN curl --retry 5 -L https://github.com/containers/podman/releases/download/v3.
     mv podman-remote-static /tools/podman-remote4 && \
     rm -f /tmp/podman-remote4.tar.gz
```
Comment (adriengentil) [inline Dockerfile.assisted-test-infra:38]: :point_up: we will likely get the same issue with `podman-remote`

**Review by osherdp — APPROVED** [review]

**Comment by adriengentil** [comment]
/hold
oups, I want to wait for a build

**Pull request merged**